addons/mrp_account/models/mrp_production.py

```diff
@@ -12,11 +12,18 @@ class MrpProduction(models.Model):
 
     extra_cost = fields.Float(copy=False, string='Extra Unit Cost')
     show_valuation = fields.Boolean(compute='_compute_show_valuation')
+    wip_production_ids = fields.Many2many('account.move')
+    wip_production_count = fields.Integer("WIP Journal Entry Count", compute='_compute_wip_production_count')
```

Comment:
Nope. No new stored field in stable. We only accept non-stored computed fields and only if there's **really** no other choice.
And if we accept the non-stored computed field, you will need to ask runbot for an exception on it to pass the ci/template checks.
Finally, how do you fill this field? I see no function or related field.

Reply:
`wip_production_ids` is the inverse of the [`wip_production_ids` field](https://github.com/odoo/odoo/blob/18.0/addons/mrp_account/models/account_move.py#L12-L15) in `account.move` under `mrp_account`, so it automatically gets populated with WIP journal entries as the other side gets populated with MOs. `wip_production_count` also mimics the same behaviour as the respective field in that model.
I know it's sinful to add a new field to stable but what the task asks for is implementing the exact inverse of the existing behaviour, so it seemed to me that this is the canonical solution. Would first merging it to master and then backporting to stable help?

Reply:
as explained in call:
- that commit can only be done in master because new stored field + new relation table + changing existing field
- the field should be written as follow: `wip_move_ids = fields.Many2many('account.move', 'wip_move_production_rel', 'production_id', 'move_id')` and its sibling field on `account.move` should be changed like this: `wip_production_ids = fields.Many2many('mrp.production', 'wip_move_production_rel', 'move_id', 'production_id')`
- merging to master first then backporting would only 'solve' part of the issues, so no

Reply:
Moved the commit to #183516.